nservicebus/azure/publisher-names-configuration.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Publishers name configuration
+summary: Configuration mapping between publisher names and message types for Endpoint Oriented Topology
+tags:
+- Cloud
+- Azure
+- Transports
+---
+
+When Endpoint Oriented Topology is selected, mapping between publisher names and message types has to be properly configured at transport level to ensure that message routing works as expected.  
+Mapping can be configured once message type at time:
```

Comment:
Mapping between publisher names and event types happens at `transport` level, instead message mapping is at `endpoint` level.
The behavior is the same. Transport needs this mapping to configure topics properly

@SeanFeldman @yvesgoeleven could you review my comment to be sure I explained `publisher mapping` feature in the right way?